housecost.js

```diff
@@ -0,0 +1,25 @@
+let volumeInMeters=8*10*10;
+let gardenSizeInM2=100;
+let peterHouseCost=2500000;
```

Comment:
here `const` is better. You don't use the same variable after to assign a new value